run/src/types.ts

```diff
@@ -27,24 +27,24 @@ export type Driver<Si, So> = {
 export type DisposeFunction = () => void;
 
 export type Drivers = {
-  [name: string]: Driver<FantasyObservable | undefined | void | null, any>;
+  [name: string]: Driver<Stream<any> | undefined, any>;
```

Comment:
The change from FantasyObservable to Stream<any> means that it's going from more general to more specialized. It may work for xstream, but we shouldn't require anything else than the "subscribe" method. Also curious what compile error happens if we revert this change. Remember that we have already released the new cycle/run with TypeScript support, so we're risking issuing yet another breaking change with this code change.

Reply:
But isnt this more correct? A driver gets and has to return xstream after all

Reply:
But isnt this more correct? A driver gets and has to return xstream after all

Reply:
Yeah I think you're right: https://cycle.js.org/releases.html#cycle-unified-for-library-authors-driver-authors